### ISSUE TITLE
Render/show subtitles over iFrame element 

### ISSUE THREAD

**Issue by hskuwari** [Dador/JavascriptSubtitlesOctopus#75]
I was wondering if you were thinking of rendering/showing .ass format subtitles over iFrame element?

**Comment by ThaUnknown** [comment]
you can do this by specifying a custom canvas which renders over the iframe? the functionality for this is there, you need to do the rest yourself

**Comment by TheOneric** [comment]
Yeah, the rendering needs a `<canvas>` object as it depends on the `CanvasRenderingContext2D` API. If you want to render over arbitrary elements, prepare an overlaying canvas, pass it to JSO and send timing updates yourself *(as there's no video time)*, see [here](https://github.com/libass/JavascriptSubtitlesOctopus/blob/1e9d5cd2147420fe1485bd1b71fbc9ee51537641/README.md#using-only-with-canvas).

**Comment by tranvuluan** [comment]
Please, help me:
My code here:
`import SubtitlesOctopus from "libass-wasm";
import React, { useState, useRef, useEffect } from "react";

const App = () => {
  const canvasRef = useRef(null);

  useEffect(() => {
    if (canvasRef.current) {
      const options = {
        canvas: canvasRef.current, // canvas element
        subUrl: '/dist/full_caption_test.ass', // Link to subtitles
        fonts: ['/fonts/arialblack.ttf'], // Links to fonts (not required, default font already included in build)
        workerUrl: '/dist/js/subtitles-octopus-worker.js', // Link to file "libassjs-worker.js"
        legacyWorkerUrl: '/dist/js/subtitles-octopus-worker-legacy.js',
        debug: true,
      };
      const instance = new SubtitlesOctopus(options);
      instance.setCurrentTime(10);
    }
  }, [canvasRef.current]);

  return (
    <div>
      <canvas ref={canvasRef}  width="600" height="600"  />
    </div>
  );
};

export default App;
`

<img width="1313" alt="image" src="https://github.com/libass/JavascriptSubtitlesOctopus/assets/63645119/37804e29-3582-4161-9dd0-8aaea7e9fb91">

But it doesn't work, 
<img width="1266" alt="image" src="https://github.com/libass/JavascriptSubtitlesOctopus/assets/63645119/8e84ef05-ea15-4716-af73-3a6ef481d9d0">
